aixplain/modules/model.py

```diff
@@ -76,6 +77,12 @@ def __init__(
         self.additional_info = additional_info
         self.url = config.MODELS_RUN_URL
         self.backend_url = config.BACKEND_URL
+        if "function" in additional_info:
+            self.function = None
```

Comment:
Add function in the docstring of model asset:

https://github.com/aixplain/aiXplain/blob/M-5038717082-ModelFilterFix/aixplain/modules/model.py#L46

Reply:
But that needs function in all caps i.e. `Function["TRANSLATION"]` will work but `Function["translation"]` will not.